src/warc2zim/main.py

```diff
@@ -198,14 +214,38 @@ def __init__(self, args):
             logger.setLevel(logging.INFO)
 
         self.indexed_urls = set({})
-        self.name = args.name
-        if not self.name:
-            self.name, ext = os.path.splitext(args.inputs[0])
-            self.name += ".zim"
+        self.output = args.output
+        if not self.output:
```

Comment:
As stated in #4, if no output is specified, it should be derived from `--name` and include the period (YYYY-MM).
What we do with other scrapers is that we have 3 args:
- `--output` (defaults to `/output`)
- `--name` mandatory
- `--zim-file` just a filename in output folder. built from `--name` if not specified

Reply:
ok, this can replace the current behavior, which uses the name of the WARC file as the name of the ZIM file by default.